tcms/templates/report/testing-report/per_build_report.html

```diff
@@ -2,23 +2,6 @@
 
 {% block report %}
 {% if report_data %}
-	{% if queries %}
-	<div class="queries">
-		<ul>
-			{% for k, v in queries.items %}
-			<li><label>{{ k|title }}:</label>{{ v|safe }};</li>
-			{% endfor %}
-			{% if not report_data.builds_selected %}
-			<li>
-				<label>Builds:</label>
-				{% for b in report_data.builds %}{{ b }}, {% endfor %}
-			</li>
-			{% endif %}
-			<li class="clear"></li>
-		</ul>
-	</div>
-	{% endif %}
```

Comment:
All templates in the testing-report directory contain the same code block. Because now queries is undefined  the if block is never executed and you don't see this HTML. However you need to remove this block from all the templates.

Reply:
Good point. I have deleted them now.